relay-server/src/actors/store.rs

```diff
@@ -374,8 +385,26 @@ impl StoreForwarder {
     }
 
     fn send_metric_message(&self, message: MetricKafkaMessage) -> Result<(), StoreError> {
+        let topic = match message.name.parse() {
```

Comment:
_nit_: This would be easier to read if you map the `namespace` here. For instance:

```suggestion
        let mri: MetricResourceIdentifier = message.name.parse();
        let topic = match mri.map(|mri| mri.namespace) {
```